tox.ini

```diff
@@ -1,22 +1,64 @@
 [tox]
 envlist =
-    py27-dj111{,-grp}
-    py{36,37}-dj{111,20}{,-grp}
+    py3{6,7,8,9}-django{22,30,31,32}
+
+[pytest]
+django_find_project = false
+DJANGO_SETTINGS_MODULE=select2.tests.settings
 
 [testenv]
-commands =
-    python runtests.py {posargs} --noinput
+description = Run tests in {envname} environment
 setenv =
-    DJANGO_SELENIUM_TESTS = 1
-passenv =
-    TRAVIS
-    TRAVIS_*
-    DATABASE_URL
+    PYTHONPATH = {toxinidir}:{toxinidir}/tests:{env:PYTHONPATH:}
+commands = {posargs:pytest}
 deps =
+    pytest>=5.2.0
+    pytest-django
     selenium
     django-selenosis
-    dj-database-url
-    dj111: Django~=1.11.0
-    dj20: Django~=2.0.0
-    dj111-grp: django-grappelli==2.10.2
-    dj20-grp: django-grappelli==2.10.2
+    django22: Django>=2.2,<3.0
+    django31: Django>=3.1,<3.2
+    django32: Django>=3.2b1,<4.0
+    django22-grp: django-grappelli==2.13.4
```

Comment:
I am not sure if these `django-grappelli` versions are correct - I just went off the releases listed [here](https://pypi.org/project/django-grappelli/)

Reply:
Should be fine, I believe we did so similarly in another package